build/toolchain/gnu/build.sh

```diff
@@ -72,6 +74,7 @@ pushd tarballs
         echo "Download gcc prerequisites..."
 
         pushd $GCC_DIRECTORY
+	    patch -Np1 -i ../../prerequisites.patch
```

Comment:
I don't see the point of this patch? What are the benefits?

Reply:
Latest release versions of dependency math libraries which are not available from the standard mirror path.